.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug Report
+description: File a bug report to help improve qBittorrent user experience.
+body:
+  - type: markdown
+    attributes:
+     value: |
+             #### ADVISORY
```

Comment:
Too much indent. Please check the indentation of the file thoroughly.

Reply:
ok